test-utils/src/main/kotlin/com/google/devtools/ksp/processor/FunctionKindProcessor.kt

```diff
@@ -27,7 +27,11 @@ open class FunctionKindProcessor : AbstractTestProcessor() {
     override fun process(resolver: Resolver): List<KSAnnotated> {
         fun KSFunctionDeclaration.pretty(): String {
             val name = if (parentDeclaration != null) {
-                val className = parentDeclaration?.simpleName?.asString() ?: ""
+                val parentClass = when (val parent = parentDeclaration) {
```

Comment:
Without that i was getting this:
```
property.getProperty: MEMBER
```
instead of this:
```
MyInterfaceImplJava.getProperty: MEMBER
```

only for KSP2. I guess it's separate issue, so I probably need to create a new one and mention it here